.github/workflows/main.yaml

```diff
@@ -79,11 +79,20 @@ jobs:
       publish_to_binaries: ${{ steps.release.outputs.publish_to_binaries }}
       release: ${{ steps.release.outputs.release }}
     steps:
+      # TODO: remove
```

Comment:
remove it ?

Reply:
As discussed per slack, I will handle it myself as you are busy the whole afternoon @matemoln